tests/Client/CosmosDbFixture.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using Azure.Identity;
+using AzureMcp.Tests.Client.Helpers;
+using Microsoft.Azure.Cosmos;
+using Xunit;
+
+namespace AzureMcp.Tests.Client;
+
+public class CosmosDbFixture : IAsyncLifetime
+{
+    private CosmosClient? _client;
+
+    public async ValueTask InitializeAsync()
```

Comment:
Can we use this Fixture with tests that already existed? It seems they use the same DB and container values.

Reply:
I made this test fixture only for cosmos, because I had to populate the database before running the new tests. All of the existing tests in `CosmosCommandTests` are already are using this one because that classes implements `IClassFixture<CosmosDbFixture>`.

Reply:
I have created this issue as a follow up to work with engsys:
https://github.com/Azure/azure-mcp/issues/345